ports/tbb/vcpkg.json

```
@@ -1,8 +1,18 @@
 {
   "name": "tbb",
-  "version-string": "2020_U3",
-  "port-version": 8,
+  "version": "2021.5.0",
   "description": "Intel's Threading Building Blocks.",
   "homepage": "https://github.com/01org/tbb",
-  "supports": "!(uwp | arm | arm64) | linux | osx"
+  "license": "Apache-2.0",
+  "supports": "!(windows & uwp) | linux | osx",
```

```suggestion
  "supports": "(windows & !uwp) | linux | osx",
```
`emscripten` passes `!(windows & uwp)`, but this is clearly an allow list that intends to exclude it.